### PR TITLE
Refactored sample apps 20.2

### PR DESCRIPTION
**Pull request by ericharmeling**
Follow-up on https://github.com/cockroachdb/docs/pull/10851, which should have been applied to 20.2 docs.

Also, removed the references to the git repo for doc versions that include local copies of the app (this should have been done beforehand).

### CODE REVIEW

**Comment by ericharmeling** [comment]
FYI @ianjevans 

Going to merge, as this PR is identical to https://github.com/cockroachdb/docs/pull/10851 (+ removing outdated repo references)

File: v20.2/build-a-python-app-with-cockroachdb-sqlalchemy.md [hunk 1]
```diff
@@ -43,133 +35,175 @@ For other ways to install SQLAlchemy, see the [official documentation](http://do
...
-It does all of the above using the practices we recommend for using SQLAlchemy with CockroachDB, which are listed in the [Best practices](#best-practices) section below.
+<section class="filter-content" markdown="1" data-scope="local">
 
-{{site.data.alerts.callout_info}}
-You must use the `cockroachdb://` prefix in the URL passed to [`sqlalchemy.create_engine`](https://docs.sqlalchemy.org/en/latest/core/engines.html?highlight=create_engine#sqlalchemy.create_engine) to make sure the [`cockroachdb`](https://github.com/cockroachdb/sqlalchemy-cockroachdb) dialect is used. Using the `postgres://` URL prefix to connect to your CockroachDB cluster will not work.
-{{site.data.alerts.end}}
+1. If you haven't already, [download the CockroachDB binary](install-cockroachdb.html).
+1. Run the [`cockroach demo`](cockroach-demo.html) command:
+
+    {% include copy-clipboard.html %}
```
Comment (ianjevans) [inline v20.2/build-a-python-app-with-cockroachdb-sqlalchemy.md:64]:
```suggestion
    {% include_cached copy-clipboard.html %}
```

File: v20.2/build-a-python-app-with-cockroachdb-sqlalchemy.md [hunk 2]
```diff
@@ -43,133 +35,175 @@ For other ways to install SQLAlchemy, see the [official documentation](http://do
...
-If you prefer, you can also clone a version of the code:
+    You'll use this connection string to connect to the database later in this tutorial.
+
+</section>
+
+## Step 3. Get the code
+
+Clone the code's GitHub repo:
 
 {% include copy-clipboard.html %}
```
Comment (ianjevans) [inline v20.2/build-a-python-app-with-cockroachdb-sqlalchemy.md:88]:
```suggestion
{% include_cached copy-clipboard.html %}
```

File: v20.2/hello-world-python-sqlalchemy.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+## Step 2. Start CockroachDB
+
+{% include {{page.version.version}}/app/start-cockroachdb.md %}
+
+## Step 3. Get the code
+
+Clone the code's GitHub repo:
+
+{% include copy-clipboard.html %}
```
Comment (ianjevans) [inline v20.2/hello-world-python-sqlalchemy.md:34]:
```suggestion
{% include_cached copy-clipboard.html %}
```

File: v20.2/hello-world-python-sqlalchemy.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+The `main` method of this program does the following:
+
+1. Attempts to connect to a running cluster, given a connection string.
+2. Prints a message to the terminal about the connection status.
+
+## Step 4. Run the code
+
+To run the app:
+
+{% include copy-clipboard.html %}
```
Comment (ianjevans) [inline v20.2/hello-world-python-sqlalchemy.md:55]:
```suggestion
{% include_cached copy-clipboard.html %}
```

**Review by ianjevans — APPROVED** [review]

**Comment by ericharmeling** [comment]
@ianjevans TFTR - apologies for rushing the merge. I've worked in your feedback in a follow-up PR: https://github.com/cockroachdb/docs/pull/10962

**Pull request merged**